format_spec/fragment.md

```diff
@@ -203,7 +203,7 @@ The fragment min max sum null count is a [generic tile](./generic_tile.md) with
 
 ### Tile Processed Conditions
 
-The processed conditions is a [generic tile](./generic_tile.md) and is the list of delete/update conditions that have already been applied for this fragment and don't need to be applied again, in no particular order, with the following internal format:
+The processed conditions is a [generic tile](./generic_tile.md) and is the list of delete/update conditions that have already been applied for this fragment and don't need to be applied again, sorted by filename, with the following internal format:
```

Comment:
If we are sorting by filename, won't the order be different than the application order?